src/core/public/public.api.md

```diff
@@ -435,6 +433,19 @@ export interface HttpErrorResponse {
     response?: Response;
 }
 
+// Warning: (ae-missing-release-tag) "HttpFetchError" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)

```

Comment:
Can you add an block comment above the definition of `HttpFetchError` like:
```
/*
 * @public 
 */
```

That will make this warning go away and mark the type as part of the public API